gradle/libraries.toml

```
@@ -30,11 +30,11 @@ kotlin-plugin-min = "1.6.0"
 kotlin-plugin = "1.9.0"
 kotlin-plugin-max = "1.9.0"
 kotlin-stdlib = "1.6.21"
-kotlinx-coroutines = "1.6.4"
+kotlinx-coroutines = "1.7.3"
 kotlinx-datetime = "0.4.0"
 kotlinx-serialization-runtime = "1.5.0"
 ksp = "1.9.0-1.0.11"
-ktor = "2.2.2"
+ktor = "2.3.2"
```

Turns out 2.3.3 was just released
```suggestion
ktor = "2.3.3"
```